config/initializers/carrierwave.rb

```diff
@@ -2,31 +2,33 @@
 
 # Default CarrierWave setup.
 #
-CarrierWave.configure do |config|
-  config.permissions = 0o666
-  config.directory_permissions = 0o777
-  config.storage = :file
-  config.enable_processing = !Rails.env.test?
+if Rails.env.production?
```

Comment:
ℹ️ Instead of hardcoding the production environment name, I also considered the possibly friendlier:

```
if Rails.aplication.secrets.aws_access_key_id.present?
```

What eventually swayed me to do Rails.env.production is that I considered what would happen if someone deleted the environment variable on production. It'd silently just start saving to the local filesystem, which is wrong. I'd want it to blow up big.